src/migrations/20210913103159-api-keys-scoping.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+          ALTER TABLE api_tokens ADD COLUMN project VARCHAR REFERENCES PROJECTS(id) ON DELETE CASCADE;; 
```

Comment:
varchar undefined length? why not text?

Reply:
Also, how did you end up solving ALL == `*` when fetching?

Reply:
> varchar undefined length? why not text?

My understanding is that you dont need length anymore. 
https://dba.stackexchange.com/a/21496

> Also, how did you end up solving ALL == * when fetching?

Yes. "*" is mapped to "null" in the database and back to "*" in the application / api layer. This way we can still leverage foreign key constraints.

Reply:
I see you're right, and I've learned something new today. Gogo